### PR TITLE
DEVDOCS-4932: [revise] Customers V3, update email `maxlimit` in customer_Post schema

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4932]

## What changed?

* Change the email `maxLimit` in Customer_Post schema
* Move requirements so that they appear on the docs page

## Anything else?

ping {names}

[DEVDOCS-4932]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by slsriehl — APPROVED** [review]

File: reference/customers.v3.yml [hunk 1]
```diff
@@ -2682,8 +2607,8 @@ components:
         email:
           description: The email of the customer. Must be unique.
           type: string
-          minLength: 3
-          maxLength: 255
+          minLength: 7
+          maxLength: 85
```
Comment (slsriehl) [inline reference/customers.v3.yml:2611]:
```suggestion
          maxLength: 250
```

**Review by slsriehl — CHANGES_REQUESTED** [review]
update to reflect limit from @bc-akshatkaushik on Customers

**Review by slsriehl — APPROVED** [review]

**Review by bc-akshatkaushik — DISMISSED** [review]
Changes look good to me 👌

**Comment by bc-tgomez** [comment]
Dropping branch as work is outdated.

**Pull request closed without merging**